contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -41,21 +41,13 @@ contract MixinStorage is
     // address for read-only proxy to call
     address public readOnlyProxyCallee;
 
-    // mapping from StakeStatus to the total amount of stake in that status for the entire
-    // staking system.
-    mapping (uint8 => IStructs.StoredBalance) public globalStakeByStatus;
```

Comment:
It could be good to keep this format for extensibility, in case we add more statuses later.

Reply:
I guess it is nice to not have to. Also makes the state layout more consistent if we do add more statuses down the line.